docs/deployment.md

```diff
@@ -0,0 +1,75 @@
+# Deploying Find - Manage Courses App - Step by Step Guide 
+
+All members of the Find development team are able to deploy into any of the environments.
+
+## 1. Check what you're deploying
+
+Go to the Find Build pipelines [Frontend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=29) & [Backend](https://dfe-ssp.visualstudio.com/Become-A-Teacher/_build?definitionId=46)  and identify the commit you want to deploy. It is advisable to make note of the build id at this stage. 
+
+Make sure to check the diff on GitHub to see if there's anything risky.
+
+You also have to make sure that you're deploying only work that has been product reviewed and approved for deployment. 
+See "To Deploy" colum on the  [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team)
+The "Product Review" column on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team) should be empty.
```

Comment:
> The "Product Review" column on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team) should be empty.

That's rarely the case in my experience

Reply:
> > The "Product Review" column on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team) should be empty.
> 
> That's rarely the case in my experience

lol @fredbrenton

Reply:
```suggestion
The "Product Review" column on the [Find Team board](https://trello.com/b/fXA6ioZN/team-board-find-team) should be empty. Any cards in that column should have their code changes merged to the `master` branch of the relevant repository, and we'll need to be sure they're safe to deploy before proceeding.
```

Reply:
@defong Why are you pulling Fred Brenton into this? 😆